week-2/errors/1.js

```diff
@@ -11,3 +11,17 @@ function convertToPercentage(decimalNumber) {
 }
 
 console.log(decimalNumber);
+
+//This will not work because in line 7 the variable decimal number declaration came after the fuction. It should come first then the function declaration.
```

Comment:
This error arises because you had a parameter `decimalNumber` defined inside your function. Then there was also a local variable defined aftwards called `decimalNumber`. Why is this a problem?

Reply:
It is a problem because we cannot declare a variable twice in one function.